snap/snapcraft.yaml

```diff
@@ -19,7 +19,7 @@ description: |
    - https://discourse.charmhub.io/
    - https://github.com/juju/juju
 
-confinement: classic
+confinement: strict
```

Comment:
this doesn't seem quite right as part of this PR

Reply:
oops, didn't mean to push that